UnityProject/Assets/Scripts/Managers/GameManager.Escape.cs

```diff
@@ -64,12 +64,6 @@ private void InitEscapeStuff()
 
 		Vector3 newPos;
 
-		if (LandingZoneManager.Instance.centcomDocking == null)
```

Comment:
Why are you removing the error log?

Reply:
And how does this fix the infinite restart?

Reply:
also it would never reset the shuttle status so it was always thinking of heading back to central command after It was being called

Reply:
So new mapping meant this was always null and returning when that didnt matter anymore?

Reply:
The object that handled this was a custom object so it didn't get saved in the new json map, so it only cropped up now